.github/workflows/main.yml

```diff
@@ -2,13 +2,9 @@ name: CI
 
 on:
   pull_request:
-    branches:
-      - master
-
   push:
     branches:
-      - master
-
+      - "**"
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
d8bd001